sklearnex/linear_model/tests/test_linear.py

```diff
@@ -75,3 +75,30 @@ def test_sklearnex_import_elastic():
     assert "daal4py" in elasticnet.__module__
     assert_allclose(elasticnet.intercept_, 1.451, atol=1e-3)
     assert_allclose(elasticnet.coef_, [18.838, 64.559], atol=1e-3)
+
+
+@pytest.mark.parametrize(
+    "dataframe,queue", get_dataframes_and_queues(dataframe_filter_="dpnp,dpctl", device_filter_="gpu")
+)
+def test_bf16_blas_epsilon(dataframe, queue):
+    from sklearnex import config_context
+    from sklearnex.linear_model import LinearRegression
+
+    size = 100
+    X = np.ones((size, 2), dtype=np.float32)
+    y = np.arange(size, dtype=np.float32) * np.finfo(np.float32).eps
+    X[:, 0] = y
+    X = _convert_to_dataframe(X, sycl_queue=queue, target_df=dataframe)
+    y = _convert_to_dataframe(y, sycl_queue=queue, target_df=dataframe)
+    linreg_standard = LinearRegression().fit(X, y)
+    with config_context(compute_mode="float_to_bf16"):
+        linreg_bf16 = LinearRegression().fit(X, y)

```

Comment:
You can offload without dpnp/dpctl dataframes, you can use config_context for that.
```suggestion
    with config_context(target_offload="gpu:0"):
        linreg_standard = LinearRegression().fit(X, y)
    with config_context(compute_mode="float_to_bf16", target_offload="gpu:0"):
        linreg_bf16 = LinearRegression().fit(X, y)
```